src/components/Candidates/CandidatesCarousel/CandidatesCarousel.css

```diff
@@ -1,12 +1,75 @@
+button:focus {
+  outline: none;
+}
+
+.candidates-carousel {
+  background: black !important;
```

Comment:
add:
height: 100vh;

to make it aprox the same height as the window

Reply:
Is this for the Small view or for large view?

Reply:
both